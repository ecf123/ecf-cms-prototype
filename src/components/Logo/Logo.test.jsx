import { customRender } from "../../utils/testUtils";

it("renders the Logo" ,()=>{
    const {container} = customRender(

    )
    expect(container).toMatchSnapshot();
});