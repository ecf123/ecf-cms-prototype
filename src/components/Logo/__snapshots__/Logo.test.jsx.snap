// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the Logo 1`] = `
<div>
  <a
    class="link"
    href="/"
  >
    <div
      class="logo"
    >
      <div
        class="logo__box"
      >
        <img
          alt="The Future Hub Logo"
          class="logo__image"
          src="ECFP-company-logo.svg"
        />
      </div>
      <p
        class="logo__top-title"
      >
        THE
      </p>
      <h3
        class="logo__title"
      >
        FUTURE HUB
      </h3>
    </div>
  </a>
</div>
`;
